### PR TITLE
feat: Workstream Protocol: Define core employee onboarding module messages[WS-25563]

### PR DESCRIPTION
**Pull request by vidhya211992**
Defined core employee onboarding module messages

### CODE REVIEW

File: src/Onboarding.proto [hunk 1]
```diff
@@ -185,3 +185,16 @@ message W4Event {
   google.protobuf.Timestamp deleted_at =16;
 }
 
+message CoreEmployeeOnboardingModuleEvent {
```
Comment (mayankdedhia) [inline src/Onboarding.proto]: Rename the event to `WSCoreEmployeeOnboardingEvent`

File: src/Onboarding.proto [hunk 2]
```diff
@@ -185,3 +185,16 @@ message W4Event {
...
+  string id=1;
+  string core_company_id=2;
+  string core_employee_id=3;
+  google.protobuf.Timestamp core_created_at=4;
+  google.protobuf.Timestamp core_updated_at=5;
+  google.protobuf.Timestamp core_deleted_at=6;
+  string status=7;
+  string type=8;
+  google.protobuf.Timestamp core_completed_at=9;
+  google.protobuf.Timestamp core_last_restarted_at=10;
```
Comment (mayankdedhia) [inline src/Onboarding.proto]: Remove `core_` from all the attributes

File: src/Onboarding.proto (hunk 1 above)
Comment (vidhya211992) [reply src/Onboarding.proto]: Done

File: src/Onboarding.proto (hunk 2 above)
Comment (vidhya211992) [reply src/Onboarding.proto]: Done

**Review by mayankdedhia — APPROVED** [review]

**Review by suyogbaldota-harbinger — APPROVED** [review]

**Review by prashant-shrikant — APPROVED** [review]

**Review by YalinZhang1 — APPROVED** [review]

**Pull request merged**